bugbug/repository.py

```diff
@@ -372,6 +378,23 @@ def _transform(commit):
 
             if size is not None:
                 source_code_sizes.append(size)
+                metrics = code_analysis_server.metrics(path, after)
+                if "error" in metrics:
+                    metrics_file_count += 1
+                    metrics = metrics["spaces"]["metrics"]
+                    commit.average_cyclomatic += metrics["cyclomatic"]
+                    commit.average_halstead_unique_operands += metrics["halstead"][
+                        "unique_operands"
+                    ]
+                    commit.average_halstead_operands += metrics["halstead"]["operands"]
+                    commit.average_halstead_unique_operators += metrics["halstead"][
+                        "unique_operators"
+                    ]
+                    commit.average_halstead_operators += metrics["halstead"][
+                        "operators"
+                    ]
+                    commit.average_source_loc += metrics["loc"]["sloc"]
```

Comment:
Is this the number of lines of code, excluding comments?

Reply:
sloc is just function span length and lloc is the number of line with something of interest.
For example, for c++:
https://github.com/mozilla/rust-code-analysis/blob/master/src/sloc.rs#L164-L178